manifests/addons/registry.yml

```diff
@@ -0,0 +1,15 @@
+---
+instance_groups:
+- name: blacksmith
+  jobs:
+  - name: registry
+    release: bosh
+  properties:
+    registry:
+      address: (( grab params.ip ))
+      host:    (( grab params.ip ))
+      db:      (( grab instance_groups.blacksmith.properties.postgres ))
+
+      username: registry-user
+      password: (( vault meta.vault "/registry:password" ))
```

Comment:
It looks like the Vault values necessary for registry aren't being generated (or asked for). A deployment on something other than vSphere causes the deploy to fail.